lib/python/ensembl/brc4/runnable/process_seq_region.py

```diff
@@ -103,7 +109,7 @@ def run(self):
         # Final file name
         metadata_type = "seq_region"
         new_file_name = metadata_type + ".json"
-        final_path = os.path.join(work_dir, new_file_name)
+        final_path = Path(work_dir, new_file_name)
```

Comment:
You can apply before the same changes I have suggested before, making `work_dir` a `Path` since the beginning.